README.md

```diff
@@ -6,22 +6,37 @@
 
 Interaction Kit is a framework for creating Discord slash command bots over the HTTP API.
 
-<!-- ## HTTP vs Gateway
+## HTTP vs Gateway
 
-In short, you probably should be using [discord.js]() unless you know why you're here.
+**TL;DR**: you probably should be using [discord.js] unless you know why you're here.
```

Comment:
Were you wanting to link this and forgot the link part of it?
`[discord.js](https://discord.js.org)`

Reply:
There's a bibliography entry at the end of the file that makes `[discord.js]` sufficient (as can be seen [here][1]), but I'll make it an inline link if Ian would prefer that.

[1]: https://github.com/stalruth/interaction-kit/tree/http-primer

Reply:
I'd vote for inline since people are more used to it